skipper.env

```diff
@@ -115,3 +115,4 @@ DEPLOY_CAPI_PROVIDER
 HOLD_INSTALLATION
 LOGGER_NAME
 AGENT_TIMEOUT_START
+ENABLE_KUBE_API
```

Comment:
seems like it is needed, otherwise the download_logs script tries to create a REST API client (and fails the complete gather step)